app.py

```diff
@@ -44,6 +44,10 @@ def index():
 def movie_cat(id):
     pass
 
+@app.route('/actors/<id>', method=['GET', 'POST'])
+def movie_cat(id):
```

Comment:
```suggestion
def actors(id):
```

Duplicate name for functions